Makefile

```
@@ -61,6 +62,7 @@ clean: ## Make a clean source tree
 	-make -C etc clean
 
 lint: ## Check code style
+	pip install pre-commit
```

Could we make this silent since it may only do something once?
```suggestion
	@pip install -q pre-commit
```